components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/step/impl/DefaultStepHandler.java

```diff
@@ -1420,4 +1427,29 @@ private void setLoggedInOrgIdInRequest(AuthenticatedIdPData authenticatedIdPData
             }
         }
     }
+
+    private void setUserOrganization(AuthenticatedUser authenticatedUser, AuthenticatorConfig authenticatorConfig) {
+
+        String userOrganization = null;
```

Comment:
```suggestion
        String userOrganization = StringUtils.EMPTY;
```
WDYT?

Reply:
Yes, that should be better. But the implementation is bit changed after the latest commit including the proposed suggession. https://github.com/wso2/carbon-identity-framework/pull/4915/commits/d0ccd16eaae1d3ee40dcd94a133037ab8036a85b